ch_backup/ch_backup.py

```diff
@@ -581,3 +584,13 @@ def _check_min_interval(self, last_backup: BackupMetadata, force: bool) -> bool:
             return True
 
         return False
+
+    def _check_schema_only_backup_skip_lock(self, sources: BackupSources) -> bool:
+        if not sources.schema_only:
+            return False
+
+        skip_lock = self._context.config.get("skip_lock_for_schema_only", None)
+        if not skip_lock:
+            return False
+
+        return skip_lock.get("backup", False)
```

Comment:
Why just `skip_lock_for_schema_only: True` is not enough, what is purpose of subnode `backup`?

Reply:
Lock is also taken in methods:
backup, restore, delete, purge, restore_access_control, fix_admin_user

Maybe later we will need to add the separate settings for this methods too.